### PR TITLE
fix: Slider step label position

### PR DESCRIPTION
**Pull request by cbunnydev**

### CODE REVIEW

File: packages/pancake-uikit/src/components/Slider/styles.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ import bunnyHeadMax from "./svg/bunnyhead-max.svg";
 import bunnyButt from "./svg/bunnybutt.svg";
 
 interface SliderLabelProps {
-  progress: number;
+  progress: string;
```
Comment (hachiojidev) [inline packages/pancake-uikit/src/components/Slider/styles.ts:9]: Why is this a string?

File: packages/pancake-uikit/src/components/Slider/Slider.tsx [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
+  StyledInput,
+  SliderLabel,
+  SliderLabelContainer,
+} from "./styles";
 import SliderProps from "./types";
 
-// We need to adjust the offset as the percentage increases, as 100% really is 100% - label width.
-// The number 10 is arbitrary, but seems to work...
-const LABEL_OFFSET = 10;
-
```
Comment (hachiojidev) [inline packages/pancake-uikit/src/components/Slider/Slider.tsx:9]: Can you explain why this was necessary to remove?

**Review by hachiojidev — CHANGES_REQUESTED** [review]

File: packages/pancake-uikit/src/components/Slider/Slider.tsx (hunk 2 above)
Comment (cbunnydev) [reply packages/pancake-uikit/src/components/Slider/Slider.tsx:9]: This is not used in the current change.

File: packages/pancake-uikit/src/components/Slider/styles.ts (hunk 1 above)
Comment (cbunnydev) [reply packages/pancake-uikit/src/components/Slider/styles.ts:9]: it can be "px" or % values. so better to use string type I think.

**Review by hachiojidev — APPROVED** [review]

**Pull request merged**